### PR TITLE
added end point to accses external server

### PR DESCRIPTION
**Pull request by sergienko4**
The original installer misses endpoint management options as in the Nuget publisher task.
I added this option as the publisher's task

### CODE REVIEW

**Comment by vtbassmatt** [comment]
Hi @sergienko4, thanks for your contribution. We expect that people will add a nuget.config containing their package sources. This way, both the build machine and on a dev box, you have access to the same sources. No API key is needed to restore packages.

File: Tasks/NuGetInstaller/task.json [hunk 1]
```diff
@@ -29,6 +30,14 @@
             "helpMarkDown": "The path to the Visual Studio solution file or NuGet packages.config",
             "required": "true"
         },
+		{
+            "name": "connectedServiceName",
+            "type": "connectedService:Generic",
```
Comment (bryanmacfarlane) [inline Tasks/NuGetInstaller/task.json]: Should there be a nuget type?  Should we do that before the task rolls out?

Comment (vtbassmatt) [inline Tasks/NuGetInstaller/task.json]: We want to do that eventually but need to put some more cycles into the design. The existing task (mis)uses generic endpoints and I'm OK with leaving it for now.

Comment (bryanmacfarlane) [inline Tasks/NuGetInstaller/task.json]: You're going to introduce a lot of compat issues and upgrade code that you will need to write.  Upgrade code that's also very expensive.

The types have extensibility and are trivial to write.

I would highly recommend holding this till that's in.  Should be quick to add.

**Pull request closed without merging**